semver/parse_range.ts

```diff
@@ -187,14 +187,12 @@ function handleGreaterThanOperator(groups: RegExpGroups): Comparator[] {
   const patch = +groups.patch;
 
   if (majorIsWildcard) return [{ operator: "<", major: 0, minor: 0, patch: 0 }];
+
   if (minorIsWildcard) {
-    if (patchIsWildcard) {
-      return [{ operator: ">=", major: major + 1, minor: 0, patch: 0 }];
-    }
-    return [{ operator: ">", major: major + 1, minor: 0, patch: 0 }];
+    return [{ operator: ">=", major: major + 1, minor: 0, patch: 0 }];
   }
   if (patchIsWildcard) {
-    return [{ operator: ">", major: major + 1, minor: 0, patch: 0 }];
+    return [{ operator: ">=", major, minor: minor + 1, patch: 0 }];
```

Comment:
See https://github.com/denoland/deno_std/issues/4344#issuecomment-1950969831

I don't think this should be `>=`